fs.go

```diff
@@ -780,6 +796,20 @@ func cleanCacheNolock(cache map[string]*fsFile, pendingFiles, filesToRelease []*
 	return pendingFiles, filesToRelease
 }
 
+func (h *fsHandler) pathToFilePath(path string) string {
+	return h.root + filepath.FromSlash(path)
```

Comment:
Shouldn't this be `filepath.FromSlash(h.root + path)` like it was in the code where you call this function now?

And then `strings.HasPrefix(filePath, h.root)` should also use `filepath.FromSlash(` I guess.

Reply:
OK. Join root and path, and then calls filepath.FromSlash https://github.com/valyala/fasthttp/pull/1331/commits/38b41a8133e7ae2c3cc0744011727482c50d033d

root and compressRoot are passed through filepath.FromSlash in normalizeRoot. (eg. `C:\path\to\root`)
The path is part of the URL, that is separated by slashes. (eg. `/path/to/file`)
The path needs filepath.FromSlash to join root. (eg. `C:\path\to\root` + `\path\to\file`)

I think the best way is to use "filepath.Join", but I used "+" and "filepath.FromSlash" to respect the original code.

```go
func (h *fsHandler) pathToFilePath(path string) string {
	return filepath.Join(h.root, path)
}

func (h *fsHandler) filePathToCompressed(filePath string) string {
	if h.root == h.compressRoot {
		return filePath
	}
	if !strings.HasPrefix(filePath, h.root) {
		return filePath
	}
	return filepath.Join(h.compressRoot, filePath[len(h.root):])
}
```